EssentialFeed/EssentialFeed/Image Comments/Image Comments Presentation/ImageCommentsPresenter.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ImageCommentsPresenter.swift
+//  EssentialFeed
+//
+//  Created by Lukas Bahrle Santana on 22/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+
+public protocol ImageCommentsView {
```

Comment:
Too many line breaks - make sure the code is consistent everywhere.

```suggestion
import Foundation

public protocol ImageCommentsView {
```